hack/dev

```diff
@@ -9,7 +9,7 @@ pushd $MAGEDIR
 go run main.go -w $DAGGER_SRC_ROOT engine:dev
 popd
 
-export DAGGER_HOST=docker-container://test-dagger-engine
+export _EXPERIMENTAL_DAGGER_CLI_BIN=$DAGGER_SRC_ROOT/bin/dagger
 export _EXPERIMENTAL_DAGGER_RUNNER_HOST=docker-container://test-dagger-engine
```

Comment:
extremely small nit: any chance we can get rid of this, since now it's all encapsulated in the CLI and we control what it does

Reply:
Yeah I considered that briefly, it would require using `-X` to overwrite the binary's default runner host (but right now -X is only used to set the version stamp, so more refactoring would be needed). I guess I'm not sure that's worth it yet since it would be a lot of lines of code to replace just this one.